api/client/client.go

```diff
@@ -5397,3 +5398,76 @@ func (c *Client) GetClusterName(ctx context.Context) (types.ClusterName, error)
 	}
 	return cn, nil
 }
+
+// HealthCheckConfigClient returns an
+// [healthcheckconfigv1.HealthCheckConfigServiceClient].
+func (c *Client) HealthCheckConfigClient() healthcheckconfigv1.HealthCheckConfigServiceClient {
+	return healthcheckconfigv1.NewHealthCheckConfigServiceClient(c.conn)
+}
+
+// GetHealthCheckConfig fetches a health check config by name.
+func (c *Client) GetHealthCheckConfig(ctx context.Context, name string) (*healthcheckconfigv1.HealthCheckConfig, error) {
```

Comment:
are these wrappers necessary?  we should avoid defining them if possible

Reply:
exactly. git server is a bad example and that's why i hope if we can find a better way.